app/DoctrineMigrations/Version20160803183748.php

```diff
@@ -31,11 +32,12 @@ public function up(Schema $schema)
     public function down(Schema $schema)
     {
         // this down() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
Sure. I did not touch any migration sql. Only generating them via console's doctrine:migrate.